testing/btest/scripts/base/frameworks/sumstats/basic-cluster.zeek

```diff
@@ -10,7 +10,7 @@
 # @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-sort btest-diff manager-1/.stdout
 
 @load base/frameworks/sumstats
-@load base/frameworks/cluster
+@load policy/frameworks/cluster/experimental
```

Comment:
I believe the more common style would be to add experimental as additional `@load` and not rely/imply transitive @loads, but cc'ing @ckreibich / @0xxon .

Sumstats also loads base/frameworks/cluster, so it was already redundant, but possibly on purpose? :man_shrugging:

Reply:
Yeah, I thought in this case it's appropriate given that experimental is a "submodule" of Cluster.

Reply:
Mostly the base vs policy difference made me stop. I've merged the version as you had it now and if there are other opinions we'll adapt.